client/js/table-view.js

```diff
@@ -18,6 +18,7 @@ class TableView{
     initDOMReferences() {
         this.headerRowEl = document.querySelector('THEAD TR');
         this.sheetBodyEl = document.querySelector('TBODY');
+        this.footie = document.querySelector('TFOOT');
```

Comment:
Please maintain consistency with the surrounding code.  `footie` is a cute name :p but inconsistent.